### ISSUE TITLE
[TS] WebdriverIO touchAction command type is wrong

### ISSUE THREAD

**Issue by vitalyiegorov** [webdriverio/webdriverio#7133]
**Environment (please complete the following information):**
 - WebdriverIO version: 7.7.4

**Describe the bug**
`packages/webdriverio/src/commands/browser/touchAction.ts` uses `...args: TouchAction[]` type which limits using all possible combination of multi action gesture as showed in the docs. for example:
```
browser.touchAction([
            { action: 'press', x: 200, y: 200 },
            { action: 'moveTo', x: 200, y: 300 },
            'release'
        ])
```

throws TS error, 

To fix this we should use `TouchActions` ('s' at the end) from `packages/webdriverio/src/types.ts` to give ability to use all possible action typing combinations: 

**To Reproduce**
Try to use the docs example for multiple touch action with TS

**Expected behavior**
No TS errors

Patch:
```
diff --git a/node_modules/webdriverio/build/commands/browser/touchAction.d.ts b/node_modules/webdriverio/build/commands/browser/touchAction.d.ts
index b0662d1..5991a67 100644
--- a/node_modules/webdriverio/build/commands/browser/touchAction.d.ts
+++ b/node_modules/webdriverio/build/commands/browser/touchAction.d.ts
@@ -1,4 +1,4 @@
-import { TouchAction } from '../../types';
+import { TouchActions } from '../../types';
 /**
  *
  * The Touch Action API provides the basis of all gestures that can be automated in Appium.
@@ -57,5 +57,5 @@ import { TouchAction } from '../../types';
  * @for android, ios
  *
  */
-export default function touchAction(this: WebdriverIO.Browser, ...args: TouchAction[]): void;
+export default function touchAction(this: WebdriverIO.Browser, ...args: TouchActions[]): void;
 //# sourceMappingURL=touchAction.d.ts.map
```

Another question, I guess this command should also return `Promise<void>` ?

**Comment by christian-bromann** [comment]
Changes would need to be applied:
- https://github.com/webdriverio/webdriverio/blob/main/packages/webdriverio/src/commands/browser/touchAction.ts#L64
- https://github.com/webdriverio/webdriverio/blob/main/packages/webdriverio/src/commands/element/touchAction.ts#L64

Would you like to help us out?

**Comment by christian-bromann** [comment]
> Another question, I guess this command should also return `Promise<void>` ?

Yes, there is no return value. Maybe `null` would be technically more accurate. Would that make a difference?

**Comment by christian-bromann** [comment]
Fixed in #7156
